packages/fxa-settings/src/lib/error-utils.ts

```diff
@@ -24,7 +24,7 @@ export interface GenericError {
   retryAfterLocalized?: string;
 }
 
-export type BeginSigninError = GenericError & {
+export type BeginSigninError = (GenericError | AuthUiError) & {
```

Comment:
I think it's preferable not to add `| AuthUiError` here so that we know `BeginSigninError` always has `errno` and `message` defined, I _think_ `GenericError` was our way of getting around that. However I know we have "// TODO: Consolidate with AuthUiError type" comment above and we have FXA-9757 filed, so eh.